packages/client/src/Conversation.tsx

```diff
@@ -244,7 +244,7 @@ function Presentable({
               onClick={() => setExpanded(!expanded)}
               aria-expanded={expanded}
               aria-label="Show more"
-              aria-controls="message-card-content"
+              aria-controls={`${presentable.id}-card-content`}
```

Comment:
Yes, this is good. In the future I recommend putting the id at the end of the string because it is very common in programming languages and markup languages to have a requirement where identifiers must begin with a letter. An HTML id is a form of identifier, and `presentable.id` happens to be a string made up of numeric characters. In fact HTML 4 did not allow ids to begin with numbers - but HTML 5 does, so this usage is ok. See https://stackoverflow.com/a/79022/103017

I only bring this up because the format of this id activated my developer spider sense because the rule that identifiers must not begin with numbers is very common.